test/abort/test_abort-aliased-buffer-overflow/binding.cc

```diff
@@ -10,7 +10,7 @@ void AllocateAndResizeBuffer(
     v8::Isolate* isolate = args.GetIsolate();
     int64_t length = args[0].As<v8::BigInt>()->Int64Value();
 
-     node::AliasedBigUint64Array array{isolate, 0};
+    node::OwningAliasedBigUint64Array array{isolate, 0};
```

Comment:
FWIW (and this is not a blocking comment for this PR) this entire file is currently dead code (it's not executed as part of our tests, see https://github.com/nodejs/node/pull/32626). I'm leaning towards removing it in a future PR (when I get around to it) as it looks like AliasedBuffer tests should be in the cctest and not tested as an addon.